pkg/server/proto/server.proto

```diff
@@ -537,6 +542,10 @@ message UI {
 
 message GetVersionInfoResponse {
   VersionInfo info = 1;
+
+  // Represents additional features that the current server supports.
+  // Useful for ensuring compatibility between clients and servers.
+  repeated string features = 4;
```

Comment:
I kept this generic - I can imagine us using this again in the future to detect feature compatibility. 

I could have just used semver checking, but I think we'll want to disable this at some point in the future.

Reply:
Should this not be an enum?

Reply:
Good idea! Done.